### PR TITLE
Add zh trans for content/en/docs/reference/glossary/disruption.md

### PR DESCRIPTION
**Pull request by ydcool**
Add zh trans for content/en/docs/reference/glossary/disruption.md

### CODE REVIEW

File: content/zh/docs/reference/glossary/disruption.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: 中断
```
Comment (tengqm) [inline content/zh/docs/reference/glossary/disruption.md:2]: disruption 统一译为“干扰”，干扰不等于中断（interruption）

Comment (ydcool) [reply content/zh/docs/reference/glossary/disruption.md:2]: got it

**Comment by sftim** [comment]
Please rebase this pull request against _master_ to bring in a work-around for issue #22222.

**Pull request merged**